circle.yml

```diff
@@ -28,9 +30,11 @@ test:
     - ./configure -i
   override:
     - env CXXFLAGS="-Werror" make test
+    - valgrind --leak-check=full --show-leak-kinds=definite,indirect,possible --error-exitcode=1 ./bin/test-libdrafter 2> memcheck.log
```

Comment:
locally you can run just `valgrind drafter apiary.apib` and you see the result so I think it is not necessary.  The flags are there to fail the CI test and have enough info in the logfile.